test/shared_setup.py

```diff
@@ -325,6 +328,7 @@ def read_setup_cfg():
 
         # realistic target
         target = {
+            'description_file': 'README.md',
```

Comment:
not sure why this is needed? we're not going to change all the setup.py to add the old default...